engine/src/main/java/org/terasology/rendering/dag/nodes/PrePostCompositeNode.java

```diff
@@ -69,16 +69,16 @@
 
     private Material prePostMaterial;
 
-    private boolean isLocalReflections;
-    private boolean isSsao;
-    private boolean isOutline;
-    private boolean isInscattering;
-    private boolean isVolumetricFog;
+    private boolean localReflectionsAreEnabled;
+    private boolean ssaoIsEnabled;
+    private boolean outlineIsEnabled;
+    private boolean inscatteringIsEnabled;
```

Comment:
My bad: I just realized the following two variables should be renamed to be easier to understand and consistent with the related nodes:
- inscatteringIsEnabled   -->  hazeIsEnabled
- setSkyBandInputTexture   -->  setHazeInputTexture

I should have thought about it earlier.